generic/tests/cfg/jumbo.cfg

```diff
@@ -4,7 +4,6 @@
     no RHEL.4
     requires_root = yes
     type = jumbo
-    mtu = 9000
     start_vm = no
     rtl8139, spapr-vlan:
         mtu = 1500
```

Comment:
So the mtu of `rtl8139` and `spapr-vlan` will always be 1500?

Reply:
After test, rtl8139 also supports mtu=9000, but spapr-vlan does not seem to belong to x86_64, so I did not find accurate results.

Reply:
So for `spapr-vlan`, the following variants will overwrite the mtu value. I think for `spapr-vlan`, the max_mtu should be 1500, and the min_mtu should same as virtio-net.

@maxujun, can you help to test it on PPC?

Reply:
sparp-valn support mtu=9000

Reply:
@maxujun Thanks for update.